documentation/modules/creating-migration-quick-plan-existing-source.adoc

```diff
@@ -22,7 +24,31 @@ The *Create migration plan* pane opens. It displays the source provider's name a
 . Click *Add mapping* to edit a suggested network mapping or a storage mapping, or to add one or more additional mappings.
 . Click *Create migration plan*.
 +
-{project-short} validates the migration plan and the *Plan details* page opens, indicating whether the plan is ready for use or contains an error. In all cases, the details of the plan are listed, and you can edit the items you filled in on the previous page. If you make any changes, {project-short} again validates the plan.
+{project-short} validates the migration plan and the *Plan details* page opens, indicating whether the plan is ready for use or contains an error. In all cases, the details of the plan are listed, and you can edit the items you filled in on the previous page. If you make any changes, {project-short} again validates the plan
+
+. VMware source providers only (All optional):
+
+* *Preserving the CPU type of VMs that are migrated from {rhv-short}*: By default, {project-short} sets the CPU type (CPU model and any flags) of migrated VMs if the type is not set specifically for the VM. This can be an issue for migrations from {rhv-short} because the CPU type of {rhv-short} VMs can be set at the cluster level, in which case {project-short} might change the type.
++
+To preserve the CPU type of your {rhv-short} VMs, in the *Settings* section, click the Edit icon next to *Preserve CPU model*. toggle the *Whether to preserve the CPU model* switch, and then click *Save*.
```

Comment:
Preserving the CPU shouldn't be under the VMware source provider?

Reply:
Fixed.